config/components/cass-operator-image-config/image_config.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: config.k8ssandra.io/v1beta1
+kind: ImageConfig
+metadata:
+  name: image-config
+images:
+  system-logger: "k8ssandra/system-logger:latest"
```

Comment:
I did a small research and it seems it's going to be tricky to avoid recopying the whole file here.

See this issue: https://github.com/kubernetes-sigs/kustomize/issues/2962.

Reply:
But this file can be created and modified with kustomize. It just can't be modified in the same kustomize build command, but one needs to execute that kustomize command twice.

Reply:
How would you create this file with kustomize?